CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On X11, fix event loop not waking up on `ControlFlow::Poll` and `ControlFlow::WaitUntil`.
 - **Breaking:** Change default `ControlFlow` from `Poll` to `Wait`.
 - **Breaking:** remove `DeviceEvent::Text`.
+- Add `Window::show_window_menu` which shows the system menu at a specified position or the current cursor position. Currently only implemented on Windows.
```

Comment:
it's very long, something like that should be enough. 

```suggestion
- Add `Window::show_window_menu` to request a titlebar/system menu; implemented on Windows for now. 
```

Reply:
a bit more clear in some cases, that it's the one attached to titlebar. One could think about macOS's app menu integrated into their bar, which is not really what it is.